src/components/cycle-section/cycle-section/CycleTopContainer.js

```diff
@@ -0,0 +1,19 @@
+import styles from './CycleSection.module.scss';
+import { Button } from 'primereact/Button';
+
+export const CycleTopContainer = ({ cycle }) => {
+  return (
+    <div className={styles.topContainer}>
+      <div>
+        <h2>Ciclo {cycle.order_number}</h2>
+        <h1>{cycle.name}</h1>
+      </div>
+      <div>
+        <Button
+          className={styles.cycleCompletitionButton}
```

Comment:
I'm not sure if we should have this button on this PR since it's not being used

Reply:
Done!